src/core/utils.js

```diff
@@ -12,12 +12,27 @@ const Stream = require('stream');
 module.exports = {
 
     lang(filePath) {
-        return fang(filePath) || {
-            name: Path.parse(filePath).ext.replace('.', '').toUpperCase(),
-            mode: 'text',
-            scope: null,
-            color: null,
-        };
+        const name = Path.parse(filePath).ext.replace('.', '').toUpperCase();
+        switch (name) {
+            case 'NUNJUCKS':
+            case 'NUNJS':
+            case 'NUNJ':
+            case 'NJ':
+            case 'J2':
```

Comment:
I guess this is missing the `njk` extension, [recommended by the Nunjucks docs](https://mozilla.github.io/nunjucks/templating.html).

Reply:
Or is it already supported by fang?

Reply:
@LeBenLeBen yes, it's in fang 2.0

Reply:
I had to revert to fang 1.0 since fang 2.0 returns a different object (no `mode` field but `ace_mode` and `codemirror_mode`. I don't want to this PR to be bigger than it needs to be, so including `NJK` is the better option.